naive_warp.py

```diff
@@ -453,23 +462,46 @@ def warp_callback(
             #     f"frontier {len(frontier)} accepted {len(acceptable)}; {len(warp_refs)} for warp"
             # )
 
-        return tuple((cmd, sum(q[1:], ())) for q in acceptable)
+        return tuple(
+            ("L", q[-1]) if _curve_is_flat(q, self._flatness) else (cmd, sum(q[1:], ()))
```

Comment:
would is_line make sense? - flat somehow throws me

Reply:
hm.. ok

Reply:
it's more like `is_almost_line`

i'm my mind "flat" conveys the concept

Reply:
+1 to is_almost_line or is_almost_linear, I think we used that style elsewhere too

Reply:
Done